Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -469,6 +534,31 @@ def pipeline_query_command(client: Client, collection: str, pipeline: str, limit
         return 'MongoDB: No results found', {}, raw_response
 
 
+def bulk_update_command(
+        client: Client,
+        collection: str,
+        filter: str,
+        update: str,
+        update_one=False,
+        upsert=False,
+        **kwargs,
+) -> Tuple[str, None]:
+
+    filter_list, update_list = parse_and_validate_bulk_update_arguments(filter, update)
+
+    response = client.bulk_update_entries(
+        collection, zip(filter_list, update_list), argToBoolean(update_one), argToBoolean(upsert)
+    )
+    if not response.acknowledged:
+        raise DemistoException('Error occurred when trying to enter update entries.')
+
+    return (
+        f'MongoDB: Total of {response.modified_count} entries has been modified.'
+        + f'\nMongoDB: Total of {response.upserted_count} entries has been inserted.',
+        None,
```

Comment:
This code uses a deprecated `return_outputs` function from CommonServerPython instead of the normal `return_results` function.
Thats why most of the command function outputs are returned as a tuple.
Fixing this requires many changes in the code.

Reply:
Great!